pybind11/__init__.py

```diff
@@ -9,10 +9,18 @@ def get_include(user=False):
     # Are we running in a virtual environment?
     virtualenv = hasattr(sys, 'real_prefix') or \
         sys.prefix != getattr(sys, "base_prefix", sys.prefix)
+    
```

Comment:
the PEP8 check fails since this empty line contains whitespaces

Reply:
Fixed